### PR TITLE
Adding Various Rules To Monitor Process Creations in Sysmon, Event Logs & EDR

### PR DESCRIPTION
**Pull request by Cyb3rEng**
Rules added were inspired ideas by :
https://github.com/vadim-hunter/Detection-Ideas-Rules/blob/main/Threat%20Intelligence/The%20DFIR%20Report/20210329_Sodinokibi_(aka_REvil)_Ransomware.yaml

Completed in contribution to TheDFIRReport: https://thedfirreport.com/2021/03/29/sodinokibi-aka-revil-ransomware/

Regards

### CODE REVIEW

**Comment by frack113** [comment]
Hello,
thanks for the work.
before I can approve please
 - check the wiki for [title](https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#title) 
 - increment '2 or 4 spaces' tags,references, list  for a better human reading
 - falsepositives use `  - unknown` but not "" 
 - author should better be `Vadim Khrykov (Theatintel) , Cyb3rEng (rule)`
 - don't use description in detection section use `# usefull informations`
 - remove `service: Sysmon` and 'selection1' as there is alreaday [windows + process_creation](https://github.com/SigmaHQ/sigma/blob/a5c6bbe04d4ca947d98079377b26ca48866e4d3f/tools/config/generic/sysmon.yml#L5)
 
I just checked the formalism.
good correction

**Comment by Cyb3rEng** [comment]
Hello, 

Sounds good to me and thank you for the feedback, I will work on it. 

Regards

**Comment by frack113** [comment]
Hi,
make some fix (convert to LF ) to pass the yaml test .
When use `category: process_creation` don't use EventID as it is add when use tools/config/generic/sysmon.yml or tools/config/generic/windows-audit.yml in the convertion. 
Check Monitor_LOLBins_Process_Creations_by_Office_applications.yml for exemple.

**Comment by Cyb3rEng** [comment]
Hello,

Thank you for pointing out those mistakes to me. Hopefully, those modifications were in line with the requirements. 

Regards

**Comment by frack113** [comment]
Hi,
Check the category : 
`process_creation` is only for eventid 1 or 4688.
EventID 11 is `file_event`  , 12-13-14 `registry_event`
View tools/config/generic/sysmon.yml for more details

**Comment by Cyb3rEng** [comment]
Hello,

thank you again for your patience. I have completed the changes to the best of my knowledge and fixed more issues that I noticed along the way. 

Regards

File: rules/windows/process_creation/Monitor_Office_Applications_Spawning_WMI_command-line.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+logsource:
+  product: windows
+  category: process_creation
+detection:
+ #useful_information: Add more office applications to the rule logic of choice
+  selection1:
+  - Image|endswith: '\wbem\WMIC.exe'
+  - ProcessCommandLine|contains: 'wmic '
+  selection2:
+  - ParentImage:
```
Comment (frack113) [inline rules/windows/process_creation/Monitor_Office_Applications_Spawning_WMI_command-line.yml:24]: Only one so no `-`

File: rules/windows/process_creation/Monitor_WMI_Win32_Process Create_command_execution_by_Office_Applications.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  product: EndPoint Detection Logs
+  category: process_creation
+detection:
+  #useful_information: Add more office applications to the rule logic of choice
+  selection1:
+    EventLog: EDR
+    EventType: WMIExecution
+    WMIcommand|contains: 'Win32_Process\:\:Create'
+  selection2:
+    - Image|endswith:
```
Comment (frack113) [inline rules/windows/process_creation/Monitor_WMI_Win32_Process Create_command_execution_by_Office_Applications.yml:25]: only one so no `-`

File: rules/windows/process_creation/Monitor_executable_and_script_files_creation_by_Office_applications_using_file_extentions.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+title: Executable and Files creation by Office Applications
```
Comment (frack113) [inline rules/windows/process_creation/Monitor_executable_and_script_files_creation_by_Office_applications_using_file_extentions.yml:1]: Title must respect [wiki](https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#title)  so `Creation`

**Review by frack113 — CHANGES_REQUESTED** [review]

**Comment by frack113** [comment]
hi,
 - attack.defence_evasion is `attack.defense_evasion` [mitre](https://attack.mitre.org/tactics/TA0005/)
 - rule must have a id unique [get one here](https://www.uuidgenerator.net/)
 - Date is `date`

File: rules/windows/process_creation/Monitor_Office_Applications_Spawning_WMI_command-line.yml (hunk 1 above)
Comment (frack113) [reply rules/windows/process_creation/Monitor_Office_Applications_Spawning_WMI_command-line.yml:24]:
```yaml
 selection2:
   ParentImage:
```

Comment (Cyb3rEng) [reply rules/windows/process_creation/Monitor_Office_Applications_Spawning_WMI_command-line.yml]: Completed fix on next commit

File: rules/windows/sysmon/Office_Applications_Spawning_WMI_command-line.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  category: process_creation
+detection:
+  #useful_information: Add more office applications to the rule logic of choice
+  selection1:
+    - Image: '\wbem\WMIC.exe'
+    - CommandLine|contains: 'wmic '
+    - OriginalFileName: 'wmic.exe'
+    - Description: 'WMI Commandline Utility'
+  selection2:
+    - ParentPrcessName|endswith:
```
[the remark this answers is not in the document]
Comment (Cyb3rEng) [reply rules/windows/sysmon/Office_Applications_Spawning_WMI_command-line.yml]: Completed fix on next commit

File: rules/windows/process_creation/Monitor_WMI_Win32_Process Create_command_execution_by_Office_Applications.yml (hunk 2 above)
Comment (Cyb3rEng) [reply rules/windows/process_creation/Monitor_WMI_Win32_Process Create_command_execution_by_Office_Applications.yml]: Completed fix on next commit

File: rules/windows/process_creation/Monitor_executable_and_script_files_creation_by_Office_applications_using_file_extentions.yml (hunk 3 above)
Comment (Cyb3rEng) [reply rules/windows/process_creation/Monitor_executable_and_script_files_creation_by_Office_applications_using_file_extentions.yml]: Completed fix on next commit

**Comment by Cyb3rEng** [comment]
Hello,

All reviews were changed and re-committed. 

Regards

**Review by frack113 — APPROVED** [review]

**Comment by frack113** [comment]
HI, there are duplicate rule

- 'rules/windows/sysmon/Monitor_LOLBins_process_creations_with_Wmiprvse_parent_process.yml',   <--to remove
- 'rules/windows/process_creation/Monitor_LOLBins_process_creations_with_Wmiprvse_parent_process.yml'

and

- 'rules/windows/sysmon/Monitor_LOLBins_Process_Creations_by_Office_applications.yml'          <--to remove
- 'rules/windows/process_creation/Monitor_LOLBins_Process_Creations_by_Office_applications.yml'

**Comment by Cyb3rEng** [comment]
hello

thank you :) , duplicate rules are now removed. 

Regards

**Pull request merged**